OracleGoldenGate/21c/bin/deployment-init.py

```diff
@@ -133,11 +133,17 @@ def get_digest_authentication():
     userName, credential = get_admin_credentials()
     return requests.auth.HTTPDigestAuth(userName, credential)
 
+def get_basic_auth_authentication():
```

Comment:
Although a nice addition, these changes are unrelated to the focus of this PR, which is startup scripts. Please open a separate issue regarding Basic/Digest authentication and create a PR for that issue separately,

Reply:
While trying to run the container each time I was seeing the error stacktrace as below on the Linux machine. Basic auth is supported by the API calls and should work absolutely fine. Fallback would be the digest authentication. I will create a separate PR for this one if that makes more sense.

`2022-06-01T13:20:23.070Z | INFO    | ALL DONE (0 errors).
Traceback (most recent call last):
  File "/usr/local/bin/deployment-init.py", line 265, in <module>
    sys.exit(main())
  File "/usr/local/bin/deployment-init.py", line 260, in main
    create_ogg_deployment()
  File "/usr/local/bin/deployment-init.py", line 251, in create_ogg_deployment
    reset_service_configuration(serviceName)
  File "/usr/local/bin/deployment-init.py", line 166, in reset_service_configuration
    config = get_service_config(serviceName)
  File "/usr/local/bin/deployment-init.py", line 140, in get_service_config
    response = get_requests_session().get(url, headers=rest_call_headers, auth=get_digest_authentication())
  File "/usr/lib/python3.9/site-packages/requests/sessions.py", line 555, in get
    return self.request('GET', url, **kwargs)
  File "/usr/lib/python3.9/site-packages/requests/sessions.py", line 542, in request
    resp = self.send(prep, **send_kwargs)
  File "/usr/lib/python3.9/site-packages/requests/sessions.py", line 662, in send
    r = dispatch_hook('response', hooks, r, **kwargs)
  File "/usr/lib/python3.9/site-packages/requests/hooks.py", line 31, in dispatch_hook
    _hook_data = hook(hook_data, **kwargs)
  File "/usr/lib/python3.9/site-packages/requests/auth.py", line 267, in handle_401
    prep.headers['Authorization'] = self.build_digest_header(
  File "/usr/lib/python3.9/site-packages/requests/auth.py", line 185, in build_digest_header
    HA1 = hash_utf8(A1)
  File "/usr/lib/python3.9/site-packages/requests/auth.py", line 148, in md5_utf8
    return hashlib.md5(x).hexdigest()
ValueError: [digital envelope routines: EVP_DigestInit_ex] disabled for FIPS `